androidClient/app/src/main/res/layout/content_about_us.xml

```diff
@@ -98,6 +140,33 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/content_about_us_fb_imgbtn" />
 
+    <TextView
+        android:id="@+id/content_about_us_morebymac_tv"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:gravity="center"
+        android:padding="8dp"
+        android:text="More By MAC"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/content_about_us_desc_tv" />
+
+    <android.widget.ListView
```

Comment:
Use recycler view

Reply:
tanmay said to use list  view

Reply:
@tanmaydixit we should use recyclerview for future customizations

Reply:
Yes i didn't think we would need to further customize, but yes a recyclerview would be better.
@Agrim1234 My bad.

Reply:
ok i will do it